libmscore/score.cpp

```diff
@@ -3027,6 +3027,11 @@ void Score::selectSimilar(Element* e, bool sameStaff)
                   pattern.subtype = -1; // hack
             else
                   pattern.subtype = e->subtype();
+            pattern.subtypeValid = true;
```

Comment:
This above line pattern.subtypeValid = true was already in Score::selectSimilarInRange(), but for some reason wasn't included here in Score::selectSimilar().  To me these two functions should behave the same when selecting subtype.  I'm guessing there was an earlier mistake which left out pattern.subtypeValid = true for selectSimilar(), which is why I've added it.  But someone make sure to double check that this makes sense.